Assets/_Laz/Scripts/UserInterface/DebugBehaviour/DebugSceneChanger.cs

```diff
@@ -0,0 +1,73 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace Laz
+{
+    public class DebugSceneChanger
+    {
+        private const string SCENE_NAME_DELIMITER = "/";
+        private const string SCENE_NAME_EXTENSION = ".unity";
+        
+        private List<string> _buildSceneNames = new List<string>();
+
+        public Action BuildSceneNamesChanged;
+
+        public List<string> BuildSceneNames => _buildSceneNames;
+
+        public void Initialize()
```

Comment:
since this just a C# class, you can just use the constructor.

Reply:
So the reason I have an Initialize method is because, in the GetBuildSceneNames method I invoke the BuildSceneNameChange Action. Which I can only subscribe to before I call Initialize(); 

Unless I pass the subscriber as a parameter to the constructor, which feels weird

Reply:
Sometimes it's not weird to pass in a subscriber as a parameter. 
it's like passing action methods through the constructor, kinda like the button

`button.addListener.onClick(someMethod)`

But this is up to you.